index.html

```diff
@@ -330,12 +330,21 @@ <h2>Introduction</h2>
 controller of an identifier, including material for cryptographic proofs and
 service endpoints for additional communications.
       </p>
-
       <p>
-It is expected that other specifications, such as [[[?DID-CORE]]], will profile
+A [=controller document=] specifies one or more
+[=verification relationships=] and [=services=] for a single, 
+identifier, for which the current controller document is taken as authoritative. 
```

Comment:
The end of this sentence does not flow well for me. Perhaps we could reorder.

```suggestion
A [=controller document=] is taken as authoritative over a single identifier for which it specifies one or more
[=verification relationships=] and [=services endpoints=].
```

Reply:
This is incorrect.  It should be zero or more service endpoints.

Reply:
Lightly tweaking @wip-abramson's suggestion, incorporating @selfissued's comment
```suggestion
A [=controller document=] is taken as authoritative over a single identifier
for which it specifies one or more [=verification relationships=] and 
zero or more [=service endpoints=].
```

Reply:
See: https://github.com/w3c/controller-document/pull/116#discussion_r1837289517

Reply:
This is outdated, but please review to see if the current language works for you @wip-abramson